internal/wasm/func_validation_test.go

```diff
@@ -36,3 +36,41 @@ func TestValidateFunction_valueStackLimit(t *testing.T) {
 		require.Equal(t, expMsg, err.Error())
 	})
 }
+
+func TestValidateFunction_FeatureSignExtensionOps(t *testing.T) {
+	// TODO: actually support, guarded by FeatureSignExtensionOps flag which defaults to false #66
+	tests := []struct {
+		input       Opcode
+		expectedErr string
+	}{
+		{
+			input:       OpcodeI32Extend8S,
+			expectedErr: "i32.extend8_s invalid as sign-extension-ops is not yet unsupported. See #66",
```

Comment:
nit: perhaps s/not unsupported/supported/?

Reply:
good idea and this is why error tests help, huh!